lib/tools/adb-emu-commands.js

```diff
@@ -36,6 +36,25 @@ emuMethods.rotate = async function () {
   await this.adbExec(['emu', 'rotate']);
 };
 
+emuMethods.sendSMS = async function (phoneNumber, message) {
+  if (_.isUndefined(phoneNumber)) {
+    log.errorAndThrow('sendSMS phoneNumber param is undefined');
+  }
+  phoneNumber = (phoneNumber+"").replace(/\s*/, "");
+  if (!/^[\+]?[(]?[0-9]*[)]?[-\s\.]?[0-9]*[-\s\.]?[0-9]{2,}$/im.test(phoneNumber)) {
```

Comment:
it would be more readable to move this pattern into a constant and give it appropriate name